sdk/src/types/block/output/nft.rs

```diff
@@ -458,6 +458,15 @@ impl StateTransitionVerifier for NftOutput {
     }
 }
 
+impl WorkScore for NftOutput {
+    fn work_score(&self, work_score_params: WorkScoreParameters) -> u32 {
+        let native_tokens_score = self.native_tokens().work_score(work_score_params);
+        let features_score = self.features().work_score(work_score_params);
+        let immutable_features_score = self.immutable_features().work_score(work_score_params);
```

Comment:
Just do the additions, no need for variables, just create unnecessary noise in situation like that IMO